pages/common/kdeconnect-cli.md

```diff
@@ -7,21 +7,21 @@
 
 `kdeconnect-cli --list-devices`
 
-- List all reachable devices:
+- List available (paired and reachable) devices:
 
 `kdeconnect-cli --list-available`
 
-- Request pairing with a specific device, specifying its ID:
+- Request pairing to a device, specifying its ID:
```

Comment:
Hrm, I think the previous description read better.

Reply:
@sbrl i find the word `specific` redundant, don't you? but seems the word `with` read better indeed

Reply:
Yeah, it is somewhat redundant, but we use _all_ over the place, and it sounds a bit firmer.